fastplotlib/utils/generate_add_methods.py

```diff
@@ -0,0 +1,67 @@
+import inspect
+import sys
+import pathlib
+
+from fastplotlib.graphics import *
+
+
+modules = list()
+
+for name, obj in inspect.getmembers(sys.modules[__name__]):
+    if inspect.isclass(obj):
+        modules.append(obj)
+
+def generate_add_graphics_methods():
+    # clear file and regenerate from scratch
+    current_module = pathlib.Path(__file__).parent.parent.resolve()
+
+    open(current_module.joinpath('layouts/graphic_methods_mixin.py'), 'w').close()
+
+    f = open(current_module.joinpath('layouts/graphic_methods_mixin.py'), 'w')
+
```

Comment:
Add a disclaimer comment to the very top of the `graphic_methods_mixin.py` file to state that this module is auto-generated and should not be modified directly